smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/validation/UnaryFunctionCall.java

```diff
@@ -1,61 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package software.amazon.smithy.typescript.codegen.validation;
-
-/**
- * For handling expressions that may be unary function calls.
- */
-public abstract class UnaryFunctionCall {
```

Comment:
this is used in JSv3, could you add it there?

Reply:
Yes, added as part of https://github.com/aws/aws-sdk-js-v3/pull/4712